src/sql/parts/query/services/queryEditorService.ts

```diff
@@ -86,6 +88,9 @@ export class QueryEditorService implements IQueryEditorService {
 				fileInput.resolve().then(m => {
```

Comment:
I think you can set the inital value on the fileinput when you create it the input.

Reply:
@anthonydresser I'm not seeing a property on the input to set for this.  Do you know which one I can use for this purpose?

Reply:
It's when you create the input
![image](https://user-images.githubusercontent.com/4324725/50994079-23ed0600-14d0-11e9-853d-45e2bb4692bb.png)